Makefile

```diff
@@ -47,6 +47,7 @@ export TAG?=latest
 IMAGE=quay.io/${REGISTRY_ORG}/troubleshooting-panel-console-plugin:${TAG}
 
 .PHONY: deploy
-deploy:				## Build and push image, deploy to cluster using help.
+deploy:				## Build and push image, reinstall on cluster using helm.
+	helm uninstall troubleshooting-panel-console-plugin -n troubleshooting-panel-console-plugin
```

Comment:
`helm uninstall` fails when there isn't already a deployment. We could move this to a "redeploy" in addition to the deploy or check if the helm chart is installed before performing the uninstall